### PR TITLE
[TRACKER-254] Refactoring getServiceResponse method HttpRequests library

### PR DESCRIPTION
**Pull request by abhinavbansal246**

### CODE REVIEW

File: src/test/java/co/cask/tracker/TestUtils.java [hunk 1]
```diff
@@ -17,8 +17,10 @@
 package co.cask.tracker;
 
 import co.cask.cdap.test.ServiceManager;
-import com.google.common.base.Charsets;
-import com.google.common.io.ByteStreams;
+import co.cask.common.http.HttpMethod;
+import co.cask.common.http.HttpRequest;
+import co.cask.common.http.HttpRequests;
```
Comment (bdmogal) [inline src/test/java/co/cask/tracker/TestUtils.java]: interesting. ``cdap-common`` is an internal module that is not exposed to apps. 

Although, this seems like test code. Can you check where ``cdap-common`` is used in tracker? If it is only used in test, make sure its a test-scope dependency?

Comment (abhinavbansal246) [inline src/test/java/co/cask/tracker/TestUtils.java]: It is being used in Tracker too apart from the test cases
https://github.com/caskdata/cask-tracker/blob/develop/src/main/java/co/cask/tracker/utils/DiscoveryMetadataClient.java#L43

**Comment by russorat** [comment]
LGTM

**Comment by russorat** [comment]
@abhinavbansal246 please squash and merge today

**Pull request merged**